index.html

```diff
@@ -0,0 +1,268 @@
+                    <h5 class="gray2 smalltext">MC Blue</h5>
+                    <h4 class="gray3">#0055B7</h4>
+                </div>
+                <div class="colorcontainer">
+                    <div class="colorcontainerbg lightbluebg">
+                    </div>
+                    <h5 class="gray2 smalltext">Light Blue</h5>
+                    <h4 class="gray3">#0196E6</h4>
+                </div>
+                <div class="colorcontainer">
+                    <div class="colorcontainerbg navybg">
+                    </div>
+                    <h5 class="gray2 smalltext">Navy Blue</h5>
+                    <h4 class="gray3">#07356A</h4>
+                </div>
+                <div class="colorcontainer">
+                    <div class="colorcontainerbg mintbg">
+                    </div>
+                    <h5 class="gray2 smalltext">Mint</h5>
+                    <h4 class="gray3">#00A499</h4>
+                </div>
+                <div class="colorcontainer">
+                    <div class="colorcontainerbg purplebg">
+                    </div>
+                    <h5 class="gray2 smalltext">Purple</h5>
+                    <h4 class="gray3">#0055B7</h4>
+                </div>
+                <div class="colorcontainer">
+                    <div class="colorcontainerbg redbg">
+                    </div>
+                    <h5 class="gray2 smalltext">Red</h5>
+                    <h4 class="gray3">#D6001C</h4>
+                </div>
+                <div class="colorcontainer">
+                    <div class="colorcontainerbg goldbg">
+                    </div>
+                    <h5 class="gray2 smalltext">Gold</h5>
+                    <h4 class="gray3">#F99D1C</h4>
+                </div>
+                <div class="colorcontainer">
+                    <div class="colorcontainerbg gray1bg">
+                    </div>
+                    <h5 class="gray2 smalltext">Dark Gray</h5>
+                    <h4 class="gray3">#2E3030</h4>
+                </div>
+                <div class="colorcontainer">
+                    <div class="colorcontainerbg gray5bg">
+                    </div>
+                    <h5 class="gray2 smalltext">Light Gray</h5>
+                    <h4 class="gray3">#F7F7F7</h4>
+                </div>
+                <div class="colorcontainer">
+                    <div class="colorcontainerbg bluegraybg">
+                    </div>
+                    <h5 class="gray2 smalltext">Blue Gray</h5>
+                    <h4 class="gray3">#B3B6BF</h4>
+                </div>
+            </div>
+        </section>
+        <section class="clearfix">
+            <h4 class="sectionheader gray3">04 Buttons</h4>
+            <div class="wrapper">
+                <div class="column30">
+                    <br>
+                    <img src="images/downcaret.svg" class="buttoniconr"><a href="#">See more</a>
+                    <br>
+                    <br>
```

Comment:
Don't use breaks to space things out. Use margin and padding. `<br>` should be retained for breaking lines of text in paragraphs, etc.